### PR TITLE
Feature function responce

### PR DESCRIPTION
**Pull request by major13ua**
* added UI view of Entity forms configuration in Admin: Form Pleayground
* added function data to form binding feature (return data from function and display in form)
* general functions is hidden in (... item) in top right corner
* added entity refresh option in (... item) in top right corner

### CODE REVIEW

File: src/app/admin/form-playground/form-playground.component.ts [hunk 1]
```diff
@@ -105,6 +138,46 @@ export class FormPlaygroundComponent implements OnInit, AfterViewInit {
...
+                startWith(''),
+                map(value => this._filterSpec(value))
+            );
+
+        /*this.spec$ = this.selectedSpecKey$.asObservable().pipe(
+            tap(console.log),
+            map(specKey => this.specs.filter(spec => spec.key === specKey).shift()),
+            // tap(spec => this.spec = spec),
+            tap(spec => this.updateXmFormTemplate(spec))
+        );*/
```
Comment (andrewkikot) [inline src/app/admin/form-playground/form-playground.component.ts]: Is it possible to remove this commented code?

File: src/app/admin/form-playground/form-playground.component.ts [hunk 2]
```diff
@@ -9,8 +9,21 @@ import {
 } from '../../shared/jsf-extention/jsf-attributes-helper';
 import { Examples } from './example-schemas.model';
 import { Principal } from '../../shared/auth/principal.service';
+import {BehaviorSubject, Observable, of, Subject} from 'rxjs';
```
Comment (andrewkikot) [inline src/app/admin/form-playground/form-playground.component.ts]: spaces

File: src/app/xm-entity/function-call-dialog/function-call-dialog.component.ts [hunk 3]
```diff
@@ -47,71 +50,96 @@ export class FunctionCallDialogComponent implements OnInit, AfterViewInit {
         // TODO think about correct way to work with context
         $.xmEntity = this.xmEntity;
         if (this.functionSpec) {
-            this.jsfAttributes = buildJsfAttributes(this.functionSpec.inputSpec, this.functionSpec.inputForm);
+            this.jsfAttributes = buildJsfAttributes(
+                this.functionSpec.inputSpec ? this.functionSpec.inputSpec : {},
```
Comment (sergeysenja1992) [inline src/app/xm-entity/function-call-dialog/function-call-dialog.component.ts]: How about use ||

File: src/app/admin/form-playground/form-playground.component.ts (hunk 2 above)
Comment (major13ua) [reply src/app/admin/form-playground/form-playground.component.ts]: fixed

File: src/app/admin/form-playground/form-playground.component.ts (hunk 1 above)
Comment (major13ua) [reply src/app/admin/form-playground/form-playground.component.ts]: fixed

File: src/app/xm-entity/function-call-dialog/function-call-dialog.component.ts (hunk 3 above)
Comment (sergeysenja1992) [reply src/app/xm-entity/function-call-dialog/function-call-dialog.component.ts]: this.functionSpec.inputSpec || {}

Comment (major13ua) [reply src/app/xm-entity/function-call-dialog/function-call-dialog.component.ts]: fixed

**Pull request merged**